pyrelational/model_managers/abstract_model_manager.py

```diff
@@ -30,11 +30,19 @@ def __init__(
         super(ModelManager, self).__init__()
 
         self.model_class = model_class
-        self.model_config = json.load(open(model_config, "r")) if isinstance(model_config, str) else model_config
+        if isinstance(model_config, str):
+            with open(model_config, "r") as f:
+                self.model_config = json.load(f)
+        else:
+            self.model_config = model_config
+
         self._current_model: Optional[E] = None
-        self.trainer_config = (
```

Comment:
Any particular reason for those changes?

Reply:
This change is relatively small, but [to follow python style guidelines](https://peps.python.org/pep-0008/) in a post pep 343 world.

The reason I tracked this down was because I was running into issues with pyrelational [or some internal feature, potentially the pytorch dataloaders themselves - I didn't fully track it down] creating a very large number of open file handles that weren't being closed which eventually led to training failures for too many open file handles. To be quite honest, I'm not convinced this was the source or not, but I left it because of the style guidelines - happy to remove if you prefer, but I don't think it's hurting.

Reply:
I don't think it's hurting either, I was just curious 🙂, happy to accept the changes!